dev/release/post-03-website.sh

```diff
@@ -22,7 +22,7 @@ set -u
 
 SOURCE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ARROW_DIR="${SOURCE_DIR}/../.."
-ARROW_SITE_DIR="${ARROW_DIR}/../arrow-site"
+: ${ARROW_SITE_DIR:="${ARROW_DIR}/../arrow-site"}
```

Comment:
Interesting, I don't know this `:=` trick.
Looks better than `=-`, only type the var once.

Reply:
Some of our shell scripts already use this idiom to set the default value. :-)